.vscode/launch.json

```diff
@@ -0,0 +1,26 @@
+{
```

Comment:
Do you want this file to be pushed? I think it is more related to your development environment right??

Reply:
add .vscode to .gitignore